db/migrate/20170126155219_add_encrypted_password_and_salt_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddEncryptedPasswordAndSaltToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :salt, :string, null: false
+    add_column :users, :encrypted_password, :string, null: false
```

Comment:
notice we'll have to first destroy all users to prevent those non-null constraints violations